pom.xml

```diff
@@ -857,15 +857,6 @@
                 <groupId>org.eclipse.jetty</groupId>
                 <artifactId>jetty-server</artifactId>
                 <version>${jetty.version}</version>
-                <exclusions>
-                    <!--
-                      Use `jakarta.servlet-api` instead.
-                      -->
-                    <exclusion>
-                        <groupId>javax.servlet</groupId>
-                        <artifactId>javax.servlet-api</artifactId>
```

Comment:
Do not remove this exclusion, `jakarta.servlet-api` is drop-in replacement of `javax.servlet-api`.

Reply:
@pan3793, in the `org.eclipse.jetty.server.Server` needs the `javax.servlet-api` denpendencies.

Reply:
I mean you can use `jakarta.servlet-api` to replace `javax.servlet-api`